docs/intentReaderHandlers/RegisterEventHandler.md

```diff
@@ -0,0 +1,342 @@
+# RegisterEventHandler 
+
+## Overview
+
+RegisterEventHandler is invoked when the task specified in the intent has the value "registerEvent". This handler is used to register for the event in FCA.
+
+It performs the following actions :
```

Comment:
Can we remove the space between actions and :

Reply:
updated for all handlers